src/features/groups/components/GroupCardList/GroupCardList.tsx

```diff
@@ -0,0 +1,13 @@
+import { IGroup } from '@/types/definition';
+
+import { GroupCard } from './GroupCard';
+
+export const GroupCardList = ({ groups }: { groups: IGroup[] }) => {
+  return (
+    <div className="px-4">
```

Comment:
How about remove this `px-4`? defining css in the parent tag is better, if not, GroupCardList and buttons will be misaligned

Reply:
And how about adding `flex flex-col gap-2` instead of `px-4`. so that we can get a correct screen to display multiple group cards.